### PR TITLE
Fix warnings in babel and clean babel client in Makefile, 

### PR DESCRIPTION
**Pull request by mislam19**
Hi,
Great tutorial.
I think these simple tweaks will make the build a bit cleaner.

### CODE REVIEW

File: babel/babel.c [hunk 1]
```diff
@@ -165,7 +165,8 @@ static ssize_t dev_read(struct file *filep, char *buffer, size_t len, loff_t *of
  */
 static ssize_t dev_write(struct file *filep, const char *buffer, size_t len, loff_t *offset){
     //  Write the string into our message memory. Record the length.
-    copy_from_user(message, buffer, len);
+    long copied_from_user;
```
Comment (dwmkerr) [inline babel/babel.c:168]: Hi @mislam19 how does this variable then get used? I notice we don't use it in the comment or code afterwards is that deliberate?

**Comment by dwmkerr** [comment]
Thanks @mislam19 just added a question on the code!

**Comment by mislam19** [comment]
this was causing a warning in the build if copy_from_user did not have a return value. We can probably log this with the pr_info / log statement in the next line. But my intent was to get rid of the warning. If you want I can just use the variable in the pr_info statement and open another pull request.

File: babel/babel.c (hunk 1 above)
Comment (mislam19) [reply babel/babel.c:168]: yes in short

**Comment by dwmkerr** [comment]
That's be great thanks @mislam19 if you have the time!

**Comment by mislam19** [comment]
close this as I am creating another one based on comments.

**Pull request closed without merging**